src/vos/vos_policy.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2021 Intel Corporation.
+ * (C) Copyright 2022 Intel Corporation.
```

Comment:
I hate to be nit-picky, but details matter when it comes to licensing stuff. You can't just replace the year with the current year -- you have to include previous years. For example, this file's copyright should be `2016-2022`. There are a couple of other instances of this problem as well.

Reply:
Just to elaborate on this, we have a commit-hook that can take the leg-work out of updating these and will apply what's needed automatically in most cases.  If you haven't installed it locally then you should, if not then an email was sent around earlier this month with the details in.

Reply:
This is a newly added file, so i guess 2022 is fine?

Reply:
> This is a newly added file, so i guess 2022 is fine?

Oh, my apologies. Yes, that's correct. I was just looking at the diff.

Reply:
> Just to elaborate on this, we have a commit-hook that can take the leg-work out of updating these and will apply what's needed automatically in most cases. If you haven't installed it locally then you should, if not then an email was sent around earlier this month with the details in.

I actually added commit-hook, hook script only works on latest commit, some of newly added files was added last year in the old commit, so i walked through files manually, obviously i missed some files unfortunately.